### PR TITLE
Add single post template

### LINKED ISSUE #12: Update single post template
Update the single post template so that it matches the [desktop](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020065%3A1078) and [mobile](https://www.figma.com/file/nf4oV51c1JBxJKXshMe7cd/New-Sensei-Theme?node-id=20020126%3A4278) designs. Example PRs [here](https://github.com/Automattic/themes/pull/6513) and [here](https://github.com/Automattic/themes/pull/6584).

We also need to ensure that the single course page looks good, since it also uses this template. Note that we don't have any designs for it, but it should look similar to a blog post, and there should be appropriate spacing between blocks.

## Desktop
Screen Shot 2022-09-30 at 11 13 14 AM

## Mobile
image copy

#### Scenario: Single post
**Given** The Sensei theme is activated
**When** I view a blog post
**Then** the blog post looks similar to the design on both desktop and mobile.

#### Scenario: Single course page
**Given** The Sensei theme is activated
**And** Sensei LMS is activated
**When** I view a single course page
**Then** the course page looks good on both desktop and mobile (no designs for this).

### PR DESCRIPTION
**Pull request by Imran92**
Implements https://github.com/Automattic/sensei-theme/issues/12

## Changes proposed

Added pattern for single post

## Testing instructions

- Create a new post, make sure the default template for post is selected
- Add a featured image to it
- Add some contents to the post
- Save and check the post from frontend
- Make sure it presents your post content in the same structure as the design

## Screenshot

Screen Shot 2022-11-03 at 12 09 27 AM

### CODE REVIEW

File: parts/header.html [hunk 1]
```diff
@@ -1,28 +1,13 @@
...
-		</div>
-		<!-- /wp:group -->
-
-		<!-- wp:navigation {"__unstableLocation":"primary","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"left","orientation":"horizontal"},"style":{"spacing":{"margin":{"top":"0"},"blockGap":"1.25rem"}}} /-->
-
-	</div>
-	<!-- /wp:group -->
-
-</div>
+<!-- wp:group {"style":{"spacing":{"padding":{"right":"20px","left":"20px"}}},"layout":{"inherit":"true","type":"constrained","contentSize":"1000px"}} -->
```
Comment (gabrielcaires) [inline parts/header.html:1]: Do we want to apply this padding on every page?  Is it working well with the other pages?

File: theme.json [hunk 2]
```diff
@@ -445,11 +446,30 @@
 					"lineHeight": "clamp(1.838rem, 1.763rem + 0.249vw, 1.95rem)",
 					"letterSpacing": "-0.01em"
 				},
+				"spacing": {
```
Comment (gabrielcaires) [inline theme.json:454]: We are using rem instead of px, should we use it here?

File: parts/header.html (hunk 1 above)
Comment (Imran92) [reply parts/header.html:1]: This is consistent with the design for the heading of all pages. If we don't put it here, we'll have to put in in all the page templates individually. That's why I thought this place will be more suitable for it

File: theme.json (hunk 2 above)
Comment (Imran92) [reply theme.json:454]: We can use px too, but as the font sizes are using rem, I thought using rem for the line height will be more suitable as that will be consistent with the font size. But I'm not sure, this is just a thought. WDYT?

**Comment by LavaGolem** [comment]
in mobile the top padding of Newsletter is off 
image

**Review by LavaGolem — APPROVED** [review]
I will approve the changes, we can address these very small comments later? 

it looks good for me in both mobile and desktop

**Comment by donnapep** [comment]
@Imran92 The PR description didn't mention testing the single course page. Was that part of the scope of this PR?

**Comment by Imran92** [comment]
> @Imran92 The PR description didn't mention testing the single course page. Was that part of the scope of this PR?

Hi @donnapep, yes I've tested with Courses too. There was an issue that you mentioned of having the blocks too tight, I've added some blockspace in this PR too to handle that.

**Pull request merged**